includes/class-wc-validation.php

```diff
@@ -79,6 +79,8 @@ public static function is_postcode( $postcode, $country ) {
 			case 'IE':
 				$valid = (bool) preg_match( '/([AC-FHKNPRTV-Y]\d{2}|D6W)[0-9AC-FHKNPRTV-Y]{4}/', wc_normalize_postcode( $postcode ) );
 				break;
+			case 'IN':
+				$valid = (bool) preg_match( '/^[1-9]{1}[0-9]{2}\s{0,1}[0-9]{3}$/', $postcode );		
```

Comment:
Seems like there are unnecessary `tab` character at the end of this line.

Reply:
@vedanshujain Since I am new to this, do I need to do anything more?

Reply:
yup, if you have time, so this PR still needs some changes. 1. Can you please add a `break;` statement after this line (like in line 81). 2. Remove the spaces at the end of this line (they might be hidden depending on your editor setting).

If you don't have the time though, then let us know, we will make those changes ourselves.

Thanks for your contribution so far!

Reply:
@vedanshujain To be honest, I am completely out of my depth now. It would be great if you finish this now! Thanks a lot!